src/main/java/edu/greenblitz/robotName/commands/CollectNote.java

```diff
@@ -11,9 +12,10 @@ public class CollectNote extends ConditionalCommand {
 
     public CollectNote() {
         super(
-                new NoteFromIntakeToShooter(),
+                new MoveArmToIntakePosition().andThen(new NoteFromIntakeToShooter()),
```

Comment:
romy - i think its better if this is parrallel or race, its nice to have the arm roller hlping but i dont want it to take too much time. make sure tho that the arm default command or smth is this position, and then do a parrallel that runs the intake and arm roller